integration_tests/test_list_09.py

```diff
@@ -1,12 +1,43 @@
 from ltypes import i32
 
 def test_list_concat():
-    x: list[i32] = [1, 2, 3]
-    y: list[i32] = [4, 5, 6]
+    x: list[i32] = []
+    y: list[i32] = []
     z: list[i32] = x + y
-
     i: i32
-    for i in range(1, 7):
+    assert len(z) == 0
+    print(z)
+
+    x = [1, 2, 3]
+    z = x + y
+    for i in range(1, 4):
+        assert z[i-1] == i
+
+    print(z)
+
+    x.clear()
+    y = [6, 7, 8]
+    z = x + y
+    for i in range(1, 4):
+        assert z[i-1] == i + 5
+    print(z)
+
+    x = [1, 2, 3, 4, 5]
+    z = x + y
+    for i in range(1, 9):
         assert z[i-1] == i
+    print(z)
+
+    x = [9, 10]
+    y = [11, 12]
```

Comment:
Please generate `x` and `y` using loops. The size should be at least 50ish so that we know things are not hard coded in the implementation.